moduleroot/.rubocop.yml.erb

```diff
@@ -6,6 +6,7 @@ defaults = {
     'rubocop-rspec',
   ],
   'AllCops' => {
+    'NewCops' => 'enable',
```

Comment:
I still think the rubocop config should not be changed here. The change isn't related to removing the changelog generator. Also I don't know if the current rubocop config handles all cops, enabling new cops could be a breaking change (that also depends on how you manage newer rubocop versions).

Reply:
done!!